netbench/netbench-driver/etc/Dockerfile

```diff
@@ -100,15 +99,18 @@ RUN set -eux; \
 FROM debian:latest
 
 ENV RUST_BACKTRACE="1"
+ARG ENDPOINT="client"
 
 # copy driver
 COPY --from=builder /app/bin /tmp/netbench
 ENV DEBIAN_FRONTEND=noninteractive
 RUN set -eux; \
-  apt-get update && apt-get install -y dnsutils curl unzip sudo; \
-  curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip"; \
-  unzip awscliv2.zip; \
-  sudo ./aws/install; \
+  if [ "$ENDPOINT" = "client" ]; then \
+    apt-get update && apt-get install -y dnsutils curl unzip sudo; \
+    curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip"; \
+    unzip awscliv2.zip; \
+    sudo ./aws/install; \
+  fi; \
```

Comment:
Why not just use the same log parsing mechanism for both the client and server?

Reply:
The CloudWatch logging method is non-ideal and I'm only using it for the server since I haven't moved in into a standalone task yet. Once that is implemented, both the client and server will just directly upload files to s3.

Reply:
hmm why is the logging method non-ideal?

Reply:
It requires extra lambda functions, setting time range parameters, creating a lot of permissions for various roles, etc. The way I handle client metrics right now is basically jsut a couple of lines in the bash script.